gym/envs/registration.py

```diff
@@ -460,6 +460,9 @@ def make(
     if isinstance(id, EnvSpec):
         spec_ = id
     else:
+        module, id = (None, id) if ":" not in id else id.split(":")
+        if module is not None:
```

Comment:
A specific error message would be nice here if this fails, we should maybe explicitly say that "You tried to create an environment via importing a package, and that import didn't work"